test/test_workspaces.py

```diff
@@ -41,3 +42,18 @@ def test_order_paths(self):
         self.assertEqual(['foo', 'bar'], order_paths(['bar', 'foo'], ['foo']))
         self.assertEqual(['baz', 'foo', 'bar'], order_paths(['bar', 'foo', 'baz'], ['baz', 'foo']))
         self.assertEqual(['foo' + os.sep + 'bim', 'bar'], order_paths(['bar', 'foo' + os.sep + 'bim'], ['foo']))
+
+    @unittest.skipUnless(sys.platform.startswith('linux'), 'requires Unix')
+    def test_order_paths_with_symlink(self):
+        try:
+            root_dir = tempfile.mkdtemp()
+            foo = os.path.join(root_dir, 'foo')
+            foo_inc = os.path.join(root_dir, 'foo/include')
```

Comment:
Nitpick: use the variable `foo`: `foo_inc = os.path.join(foo, 'include')`

Reply:
Valid remark, considering how tested algorithm avoids using naked '/' - done.